examples/democoin/x/oracle/handler.go

```diff
@@ -0,0 +1,101 @@
+package oracle
+
+import (
+	"bytes"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Handler handles payload after it passes voting process
+type Handler func(ctx sdk.Context, p Payload) sdk.Error
+
+func (keeper Keeper) update(ctx sdk.Context, val sdk.Validator, valset sdk.ValidatorSet, p Payload, info Info) Info {
+	info.Power = info.Power.Add(val.GetPower())
+
+	totalPower := valset.TotalPower(ctx)
+	if !info.Power.GT(totalPower.Mul(keeper.supermaj)) {
```

Comment:
Maybe separate `totalPower.Mul(keeper.supermaj)` to `requiredPower` variable for clarity?